scripts/run_daily_real_validation.py

```
@@ -0,0 +1,1410 @@
+        raise argparse.ArgumentTypeError("resolution width and height must be positive")
+    return width, height
+
+
+def resolution_to_dict(resolution: tuple[int, int] | None) -> dict[str, int] | None:
+    if resolution is None:
+        return None
+    return {"width": int(resolution[0]), "height": int(resolution[1])}
+
+
+def _run_git(args: list[str], cwd: Path) -> str:
+    completed = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)